launchable/commands/record/session.py

```diff
@@ -73,16 +74,23 @@ def session(
     """
 
     flavor_dict = {}
-    for f in normalize_flavors(flavor):
+    for f in normalize_key_value_types(flavor):
         flavor_dict[f[0]] = f[1]
 
-    link = _capture_link(os.environ)
     payload = {
         "flavors": flavor_dict,
         "isObservation": is_observation,
     }
-    if link:
-        payload["link"] = link
+
+    _links = capture_link(os.environ)
+    if len(links) != 0:
+        for link in normalize_key_value_types(links):
+            _links.append({
+                "title": link[0],
+                "url": link[1],
+                "kind": "CUSTOM_LINK"
```

Comment:
[nits]
Probably better to use `LinkKind.CUSTOM_LINK`.
https://github.com/launchableinc/cli/pull/529/files#diff-280a9bc4f9d9c38c29d666a70adf8c4e216a1abb22f548b3a011f9ac00b6bf10R18